tests/unit/utils/toHoursAndMinutes.spec.js

```diff
@@ -0,0 +1,19 @@
+import toHoursAndMinutes from "@/utils/toHoursAndMinutes";
+
+const input = 71;
+const output = toHoursAndMinutes(input);
+
+describe("toHoursAndMinutes", () => {
+  it("returns value", () => {
+    expect(output).toBeTruthy;
+  });
+
+  it("returns string", () => {
+    expect(typeof output).toBe("string");
+  });
+
+  it("return string in correct format", () => {
+    const re = /\dh\s\d{2}min/;
+    expect(output).toMatch(re);
```

Comment:
tbh. it's much easier to understand the function itself that the test - it should be the other way around 😉

Reply:
when I step back and look at this again now that youve pointed it out - I really need to be thinking from the third's person perspective when writing tests 😅 NOTED 🗒️ ✔️